index.html

```diff
@@ -6,15 +6,59 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <link rel="stylesheet" type="text/css" href="css/grid.css">
+    <link rel="stylesheet" type="text/css" href="css/main.css">
+
+    <link href="https://fonts.googleapis.com/css?family=Open+Sans&display=swap" rel="stylesheet">
+    
+
     <!-- Add a link to your custom CSS here -->
 
     <title>Responsive grid project</title>
-</head>
+    </head>
 
-<body>
+<body class="sm-col-12 ">
 
```

Comment:
Remember when implementing the 12 grid system to first have a html element with class set to "container", a child element then set to "row" (so that flexbox is implemented) and then use "col-" classes for your child elements.